nilearn/plotting/displays.py

```diff
@@ -415,8 +415,7 @@ def _add_markers(self, marker_coords, marker_color, marker_size, **kwargs):
                     len(marker_color) != 1:
                 marker_color = marker_color[relevant_coords]
             
-            if isinstance(marker_size, np.ndarray) or \
-                isinstance(marker_size, list):
+            if not isinstance(marker_size, numbers.Number):
                 marker_size = marker_size[relevant_coords]
```

Comment:
In the fix you suggested in #2185 you made sure `marker_size` got converted to a numpy array when it is a list, don't we still need that?

Reply:
Yes, I was thinking about that. In the docs of ```plot_connectome``` the ```marker_size``` is a defined as scalar or array_like, so I supposed that it is not a list.
BTW I can edit it quickly.